src/test/java/org/opensearch/ad/ml/CheckpointDaoTests.java

```diff
@@ -1067,27 +1067,22 @@ public void testDeserializeTRCFModel() throws Exception {
         coldStartData.add(sample4);
         coldStartData.add(sample5);
 
-        // This scores were generated with the sample data but on RCF3.0-rc1 and we are comparing them
-        // to the scores generated by the imported RCF3.0-rc2.1
+        // This scores were generated with the sample data on RCF4.0. RCF4.0 changed implementation
+        // and we are seeing different rcf scores between 4.0 and 3.8. This is verified by switching
+        // rcf version between 3.8 and 4.0 while other code in AD unchanged. But we get different scores.
         List<Double> scores = new ArrayList<>();
-        scores.add(4.814651669367903);
-        scores.add(5.566968073093689);
-        scores.add(5.919907610660049);
-        scores.add(5.770278090352401);
-        scores.add(5.319779117320102);
-
-        List<Double> grade = new ArrayList<>();
-        grade.add(1.0);
-        grade.add(0.0);
-        grade.add(0.0);
-        grade.add(0.0);
-        grade.add(0.0);
+        scores.add(5.052069275347555);
```

Comment:
Do you think we should keep the old scores to do have a record of the comparison as we upgrade RCF or that is not really relevant cause point is to test existing functionality, old scores can be seen in commit history basically?

Reply:
I think the latter. We can find the old scores from commit history.